### PR TITLE
Add tuiview

### PR DESCRIPTION
**Pull request by gillins**
http://tuiview.org/

I noticed there doesn't seem to be any viewers for remotely sensed data, so I've added this lightweight one (I'm main author so a bit biased). Let me know if not suitable.

I'm not sure if the gdal version needs to be pinned like we did for RIOS?

### CODE REVIEW

File: recipes/tuiview/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+package:
+    name: tuiview
+    version: {{ version }}
+
+source:
+    fn: TuiView-{{ version }}.tar.gz  # [unix]
+    fn: TuiView-{{ version }}.zip  # [win]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.tar.gz  # [unix]
+    url: https://bitbucket.org/chchrsc/tuiview/downloads/TuiView-1.1.8.zip  # [win]
```
Comment (ocefpaf) [inline recipes/tuiview/meta.yaml]: Do you need the two sources due to some line endings difference or something like that?

File: recipes/tuiview/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python setup.py install --prefix=$PREFIX
```
Comment (ocefpaf) [inline recipes/tuiview/build.sh]: Just to be on the safe side can you change this to:

```shell
$PYTHON setup.py install
```

PS: Not sure you really need the `$PREFIX`

**Comment by ocefpaf** [comment]
@gillins I had no idea that `tuiview` existed! I am teaching a GIS course next month and that will be very useful!!

**Comment by ocefpaf** [comment]
I guess you need set the GDAL var for OS X too.

```shell
+ python setup.py install --prefix=/Users/travis/miniconda/envs/_build
GDAL with Python bindings must be installed first
Command failed: /bin/bash -x -e /Users/travis/build/conda-forge/staged-recipes/recipes/tuiview/build.sh
```

File: recipes/tuiview/meta.yaml (hunk 1 above)
Comment (msarahan) [inline recipes/tuiview/meta.yaml]: Unless they include different files, line endings don't matter.  You can keep just the tarball and things should work

Comment (msarahan) [inline recipes/tuiview/meta.yaml]: Also, use {{ version }} in the URL

Comment (gillins) [inline recipes/tuiview/meta.yaml]: Problem was that the .zip file (created using sdist on Windows) contains the .bat files, whereas the .tar.gz (from Linux) does not. See https://bitbucket.org/chchrsc/tuiview/src/1a17bc21d8064ce5510d654efb9e1400a5140709/setup.py?fileviewer=file-view-default#setup.py-111. 
Maybe I should just be using the .zip file for both platforms?
Is supplying a .bat file so TuiView can be easily started from the command line still Conda practice?

**Comment by gillins** [comment]
@ocefpaf thanks and I hope it is handy - I haven't been able to find anything else out there that gives you proper access to the Raster Attribute Table which is essential for proper GIS. We did have to make some modifications to GDAL first (see https://trac.osgeo.org/gdal/wiki/rfc40_enhanced_rat_support)!

File: recipes/tuiview/meta.yaml (hunk 1 above)
Comment (msarahan) [inline recipes/tuiview/meta.yaml]: bat files are fine if they're necessary, but they're usually not.  If your code runs a particular Python file or function, you can use entry points instead: http://conda.pydata.org/docs/building/meta-yaml.html#python-entry-points

zip file for both platforms is fine, IMHO.  Not a huge issue, though.

**Comment by gillins** [comment]
@msarahan turns out the code wasn't clean enough for using python entry points, but I've created an issue (https://bitbucket.org/chchrsc/tuiview/issues/3/make-tuiview-and-tuiviewwritetable) and hopefully get this right next release. I've gone with the .zip file now on all platorms.

@ocefpaf That error on OSX is weird - happens when it can't `from osgeo import gdal`. I've tested on the Mac and it seems fine. Hopefully it works this time...

**Comment by gillins** [comment]
OK on the Mac I'm getting this error:
```
libgdal.20.dylib requres version 38.0.0 or later, but libpng16.16.dylib provides verision 34.0.0
```
This only happens in the build environment, not otherwise. I'm very confused... Any ideas?

**Comment by ocefpaf** [comment]
> This only happens in the build environment, not otherwise. I'm very confused... Any ideas?

`libpng` was downgraded to accommodate  newer `libnetcdf` on OS X. I guess that my tricks\* to fix other libraries reflect here and all the builds are passing.

\* I had to remove some old builds that were not pinned and were somehow preferred by conda.

File: recipes/tuiview/build.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --prefix=$PREFIX
```
Comment (ocefpaf) [inline recipes/tuiview/build.sh]: I don't think you need to specify `--prefix=$PREFIX` .

**Pull request merged**